teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization, _name = 'Unknown')
+    super(4, true)
```

Comment:
- [ ] As I can see the parameter here is missing the other since you inherit it from person so you should  add person parameter as well with their defualt values so kindly add them with the initialize and with the super 🔥💯🚀